### PR TITLE
#107 Add the posibility to display the "ReqIF.Text" attribute of Requirement elements in Capella Richtext editor

### LINKED ISSUE #107: Add the posibility to display the "ReqIF.Text" attribute of Requirement elements in Capella Richtext editor
Currently, the "ReqIF.Text" field is in plain text mode eventhough it has information to be displayed in a Richtext editor. The idea is to add the posibility to display the "ReqIF.Text" attribute of Requirement elements in Capella Richtext editor.

### PR DESCRIPTION
**Pull request by minhtutonthat**
Resolves #107

Change-Id: Ie75a86317b6ddcc53da7047839e4a1be99de06c7
Signed-off-by: Tu Ton <minhtutonthat@gmail.com>

### CODE REVIEW

File: plugins/org.polarsys.capella.vp.requirements.importer/src/org/polarsys/capella/vp/requirements/importer/transposer/bridge/ImageImporter.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ *    Thales - initial API and implementation
+ *******************************************************************************/
+package org.polarsys.capella.vp.requirements.importer.transposer.bridge;
+
+public class ImageImporter {
+  private ImageImportStrategy imgImportStragety;
+  private String absPath;
+  private String relPath;
+  
+  public ImageImportStrategy getImgImportStragety() {
```
Comment (pdulth) [inline plugins/org.polarsys.capella.vp.requirements.importer/src/org/polarsys/capella/vp/requirements/importer/transposer/bridge/ImageImporter.java:20]: ImportStrategy

File: plugins/org.polarsys.capella.vp.requirements.importer/src/org/polarsys/capella/vp/requirements/importer/transposer/bridge/ImageImporter.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    return imgImportStragety;
+  }
+  
+  public void setImgImportStragety(ImageImportStrategy imgImportStragety) {
```
Comment (pdulth) [inline plugins/org.polarsys.capella.vp.requirements.importer/src/org/polarsys/capella/vp/requirements/importer/transposer/bridge/ImageImporter.java:24]: ImportStrategy

File: plugins/org.polarsys.capella.vp.requirements.importer/src/org/polarsys/capella/vp/requirements/importer/transposer/bridge/ImageImportStrategy.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+ * http://www.eclipse.org/legal/epl-2.0
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ *  
+ * Contributors:
+ *    Thales - initial API and implementation
+ *******************************************************************************/
+package org.polarsys.capella.vp.requirements.importer.transposer.bridge;
+
+public enum ImageImportStrategy {
```
Comment (pdulth) [inline plugins/org.polarsys.capella.vp.requirements.importer/src/org/polarsys/capella/vp/requirements/importer/transposer/bridge/ImageImportStrategy.java:15]: ImportStrategy

File: plugins/org.polarsys.capella.vp.requirements.importer/src/org/polarsys/capella/vp/requirements/importer/transposer/bridge/ImageImportingDialog.java [hunk 4]
```diff
@@ -0,0 +1,249 @@
...
+public class ImageImportingDialog extends TitleAreaDialog {
+
+  private Text absPathText;
+  private Button absPathChoosingButton;
+  private Text relativePathText;
+  private Button relativePathChoosingButton;
+  private Button embedImageButton;
+  private ImageImporter imageImporter;
+  private IProject currentProject;
+  private static final String DEFAULT_MESSAGE = "The imported ReqIf file references images. Please indicate below how to import these images into your model.\n";
```
Comment (pdulth) [inline plugins/org.polarsys.capella.vp.requirements.importer/src/org/polarsys/capella/vp/requirements/importer/transposer/bridge/ImageImportingDialog.java:44]: can you externalize string to messages?

**Review by pdulth — CHANGES_REQUESTED** [review]

File: plugins/org.polarsys.capella.vp.requirements.importer/src/org/polarsys/capella/vp/requirements/importer/transposer/bridge/ImageImporter.java (hunk 1 above)
Comment (minhtutonthat) [reply plugins/org.polarsys.capella.vp.requirements.importer/src/org/polarsys/capella/vp/requirements/importer/transposer/bridge/ImageImporter.java:20]: Nice catch :)

**Review by pdulth — APPROVED** [review]

**Pull request merged**